src/System.Private.ServiceModel/src/Extensions/ReflectionExtensions.cs

```diff
@@ -92,7 +92,7 @@ public static bool IsValueType(this Type type)
         }
         public static InterfaceMapping GetInterfaceMap(this Type type, Type interfaceType)
         {
-            throw ExceptionHelper.PlatformNotSupported();
+            return type.GetTypeInfo().GetRuntimeInterfaceMap(interfaceType);
```

Comment:
Yep - this is a change I made while working on the test+product earlier. 
This change allows us to look for the interface(s) implemented by a given type.